apps/web/src/components/workflow/FlowEditor.tsx

```diff
@@ -88,23 +84,24 @@ export function FlowEditor({
 }: IFlowEditorProps) {
   const { colorScheme } = useMantineColorScheme();
   const reactFlowWrapper = useRef<HTMLDivElement>(null);
-  const [nodes, setNodes, onNodesChange] = useNodesState(initialNodes);
-  const [edges, setEdges, onEdgesChange] = useEdgesState<IEdge>(initialEdges);
-  const [reactFlowInstance, setReactFlowInstance] = useState<ReactFlowInstance>();
-  const { setViewport } = useReactFlow();
+  const [nodes, setNodes, onNodesChange] = useNodesState([]);
+  const [edges, setEdges, onEdgesChange] = useEdgesState<IEdge>([]);
+  const reactFlowInstance = useReactFlow();
   const { readonly } = useEnvController();
   const [displayEdgeTimeout, setDisplayEdgeTimeout] = useState<Map<string, NodeJS.Timeout | null>>(new Map());
 
   useEffect(() => {
     const clientWidth = reactFlowWrapper.current?.clientWidth;
     const middle = clientWidth ? clientWidth / 2 - 100 : 0;
     const zoomView = 1;
-    reactFlowInstance?.project({ x: middle, y: 10 });
-    setViewport({ x: middle, y: 10, zoom: zoomView });
+
+    reactFlowInstance.setViewport({ x: middle, y: 10, zoom: zoomView });
   }, [reactFlowInstance]);
 
   useEffect(() => {
-    initializeWorkflowTree();
+    setTimeout(() => {
+      initializeWorkflowTree();
+    }, 0);
```

Comment:
After the viewport is updated we render the nodes and edges. This way we are skipping the initial blink that can be noticed here:

https://github.com/novuhq/novu/assets/2607232/3b2449e9-a1ec-4460-9b93-1f5e29f850e8